config/environments/development.rb

```diff
@@ -26,6 +26,11 @@
     config.cache_store = :null_store
   end
 
+  # Use a real queuing backend for Active Job (and separate queues per environment)
+  config.active_job.queue_adapter     = :resque
+  # config.active_job.queue_name_prefix = "glowfic_#{Rails.env}"
+  config.action_mailer.perform_caching = false
+
```

Comment:
Development should also queue to Resque (otherwise I think it runs jobs inline).

Reply:
We might want to look into `async` as `queue_adapter` for development: <http://api.rubyonrails.org/classes/ActiveJob/QueueAdapters.html#module-ActiveJob::QueueAdapters-label-Async+and+Inline+Queue+Adapters>